packages/create-cloudflare/.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
+node_modules
```

Comment:
Perhaps as a cleanup later on, but these are already covered in the top level .gitignore, and generally we have been trying to avoid adding further .gitignores to the monorepo. For example the top-level one contains specific ignores for packages/wrangler, packages/prerelease-registry and packages/wranglerjs-compat-webpack-plugin - although I note that wrangler-devtools sneaked its own file in recently...

Reply:
Will remove then.